docs/tasks/configure-pod-container/dapi-envars-pod.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dapi-envars-fieldref
+spec:
+  containers:
+    - name: test-container
+      image: gcr.io/google_containers/busybox
+      command: [ "sh", "-c"]
+      args: ["while true; do echo -en '\n'; \
```

Comment:
Why not use the multiline yaml syntax.  I think it is something like this:

args: |
  while true; do
    echo -en '\n';
    printenv MY_NODE_NAME MY_POD_NAME MY_POD_NAMESPACE \
                   MY_POD_IP MY_POD_SERVICE_ACCOUNT;
    sleep 10;
    done